content/docs/installation.md

```diff
@@ -4,15 +4,129 @@ description: Get up and running on Linux and a variety of container platforms
+```
+
+6. With the dependencies of the configmap created, the daemonset can now be created.
+```shell
+kubectl apply -f k8s-with-rbac/falco-daemonset-configmap.yaml
+```
+
+7. In order to confirm that Falco started correctly. You can check the status of the Falco pods by checking the logs.
+```shell
+kubectl logs -l app=falco
+```
+
+### Minkube
+
+The easiest way to use Falco on Kubernetes in a local environment is on [Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/). Both the Kubernetes YAML manifests, and the Helm chart are regularly tested with Minikube.
+
+#### Minikube Kernel Module
+
+When running `minikube` with the default `--driver` arguments, Minikube creates a VM that runs the various Kubernetes services and a container framework to run Pods, etc. Generally, it's not possible to build the Falco kernel module directly on the Minikube VM, as the VM doesn't include the kernel headers for the running kernel.
+
+To address this, starting with falco 0.13.1 we pre-build kernel modules for the last 10 Minikube versions and make them available at https://s3.amazonaws.com/download.draios.com. This allows the download fallback step to succeed with a loadable kernel module.
+
+Going forward, we'll continue to support the most recent 10 Minikube versions with each new Falco release. We also keep previously built kernel modules around for download, so we will continue to have limited historical support as well.
+
+### GKE
+
+Google Kubernetes Engine's (GKE) default operating system for worker node pools is Container-Optimized OS (COS), a security enhanced operating system that limits access to certain parts of the underlying OS. Because of these security enhancements, Falco cannot insert its kernel module inorder to process events for system calls. COS does provide the ability to leverage eBPF (extended Berkeley Packet Filter) to provide the stream of system calls to the Falco engine. eBPF is currently only supported on GKE and COS.
+
+#### Enabling eBPF Support
+
+Falco can be configured to use eBPF by setting the environment variable `SYSDIG_BPF_PROBE` to an empty value (ie `SYSDIG_BPF_PROBE=""`). When this environment variable is set, the `falco-probe-loader` script will attempt to download the kernel headers for the appropriate version of COS, and then compile the appropriate eBPF probe. Alternatively, you can set `SYSDIG_BPF_PROBE` to the path of an existing eBPF probe.
+
+If using Helm, you can enable eBPF by setting the `ebpf.enable` configuration option.
+```shell
+helm install --name falco stable/falco --set ebpf.enabled=true
+```
+
+If using the provided daemonset manifests, you can uncomment the following lines in the daemonset file.
+```
+          env:
+          - name: SYSDIG_BPF_PROBE
+            value: ""
+```
 
 ## Linux
 
+Falco can be installed directly on Linux via a scritped install, package managers, or configuration management tools like Ansible. Installing Falco directly on the host provides the following:
```

Comment:
Install Falco directly on Linux via a scripted install, package managers, or configuration management tools like Ansible. Installing Falco directly on the host provides: